sys/ztimer/Makefile.include

```diff
@@ -12,4 +14,10 @@ ifneq (,$(filter ztimer_periph_rtt,$(USEMODULE)))
     RTT_FREQUENCY ?= RTT_MAX_FREQUENCY
     CFLAGS += -DRTT_FREQUENCY=$(RTT_FREQUENCY)
   endif
+
+  MODULES_ZTIMER_ON_RTT_CONFLICTING = $(filter $(MODULES_ZTIMER_ON_RTT_CONFLICT),$(USEMODULE))
+  ifneq (0,$(words $(MODULES_ZTIMER_ON_RTT_CONFLICTING)))
+    $(info $(COLOR_YELLOW)WARNING! The following modules conflict with 'ztimer_periph_rtt': '$(MODULES_ZTIMER_ON_RTT_CONFLICTING)')
+    $(info To disable ztimer periph_rtt auto-inclusion add 'ztimer_no_periph_rtt' to 'USEMODULE'$(COLOR_RESET))
```

Comment:
```
$ BOARD=nucleo-f767zi USEMODULE='ztimer_periph_rtt ztimer_no_periph_rtt' make info-modules
WARNING! The following modules conflict with 'ztimer_periph_rtt': 'ztimer_no_periph_rtt'
To disable ztimer periph_rtt auto-inclusion add 'ztimer_no_periph_rtt' to 'USEMODULE'
...
ztimer_no_periph_rtt
ztimer_periph_rtt
ztimer_periph_timer
...
```

Reply:
Having another think about this: I am sure that this warning should not betriggert since `ztimer_no_periph_rtt` does not override `ztimer_periph_rtt`

Reply:
It does not but if you include it in the wrong place (not application or BOARD level) then you would get a warning. But Ok will remove.